app/ts/library/utilities.ts

```diff
@@ -64,3 +64,15 @@ export function areEqualStrings(a: string, b: string, caseSensitive?: true) {
 	if (caseSensitive) return a === b
 	return a.toLowerCase() === b.toLowerCase()
 }
+
+/**
+ * Abbreviate huge numbers
+ */
+export function abbreviateNumber(num: number): string {
+	const suffixes = ["", "K", "M", "B", "T", "Q"]
```

Comment:
```suggestion
	const suffixes = ['', 'k', 'M', 'G']
```
let's use SI units and not use the too big ones as they are not very familiar to people